webcommon/libs.graaljs/manifest.mf

```diff
@@ -1,7 +1,7 @@
 Manifest-Version: 1.0
-AutoUpdate-Show-In-Client: false
+AutoUpdate-Show-In-Client: true
 OpenIDE-Module: org.netbeans.libs.graaljs/1
 OpenIDE-Module-Layer: org/netbeans/libs/graaljs/layer.xml
 OpenIDE-Module-Localizing-Bundle: org/netbeans/libs/graaljs/Bundle.properties
 OpenIDE-Module-Specification-Version: 1.7
```

Comment:
Increase module version when you are changing the API (by increasing set of provided tokens).